### PR TITLE
Release Zebra 1.0.0-rc.3

### LINKED ISSUE #5785: Release Zebra 1.0.0-rc.3
## Motivation

We do regular Zebra releases.

### PR DESCRIPTION
**Pull request by teor2345**
Closes #5785

### How to Increment Versions

Zebra follows [semantic versioning](https://semver.org).
Semantic versions look like: MAJOR`.`MINOR`.`PATCH[`-`TAG`.`PRE-RELEASE]

The [draft `zebrad` changelog](https://github.com/ZcashFoundation/zebra/releases) will have an automatic version bump. This version is based on [the labels on the PRs in the release](https://github.com/ZcashFoundation/zebra/blob/main/.github/release-drafter.yml).

Check that the automatic `zebrad` version increment is correct:

If we're releasing a mainnet network upgrade, it is a `major` release:
1. Increment the `major` version of _*all*_ the Zebra crates.
2. Increment the `patch` version of the tower crates.

If we're not releasing a mainnet network upgrade, check for features, major changes, deprecations, and removals. If this release has any, it is a `minor` release:
1. Increment the `minor` version of `zebrad`.
2. Increment the `pre-release` version of the other crates.
3. Increment the `patch` version of the tower crates.

Otherwise, it is a `patch` release:
1. Increment the `patch` version of `zebrad`.
2. Increment the `pre-release` version of the other crates.
3. Increment the `patch` version of the tower crates.

Zebra's Rust API is not stable or supported, so we keep all the crates on the same beta `pre-release` version.

### Version Locations

Once you know which versions you want to increment, you can find them in the:

zebrad (rc):
- [x] zebrad `Cargo.toml`
- [x] `zebra-network` protocol user agent: https://github.com/ZcashFoundation/zebra/blob/main/zebra-network/src/constants.rs
- [x] `README.md`
- [x] `book/src/user/docker.md`

crates (pre-release):
- [x] zebra-* `Cargo.toml`s

tower (patch):
- [x] tower-* `Cargo.toml`s

auto-generated:
- [x] `Cargo.lock`: run `cargo build` after updating all the `Cargo.toml`s

#### Version Tooling

You can use `fastmod` to interactively find and replace versions.

For example, you can do something like:
```
fastmod --extensions rs,toml,md --fixed-strings '1.0.0-rc.0' '1.0.0-rc.1' zebrad README.md zebra-network/src/constants.rs
fastmod --extensions rs,toml,md --fixed-strings '1.0.0-beta.15' '1.0.0-beta.16' zebra-*
fastmod --extensions rs,toml,md --fixed-strings '0.2.30' '0.2.31' tower-batch tower-fallback
```

If you use `fastmod`, don't update versions in `CHANGELOG.md` or `zebra-dependencies-for-audit.md`.

## README

Update the README to:
- [x] Remove any "Known Issues" that have been fixed
- [x] Update the "Build and Run Instructions" with any new dependencies.
      Check for changes in the `Dockerfile` since the last tag: `git diff <previous-release-tag> docker/Dockerfile`.

## Checkpoints

With every release and for performance reasons, we want to update the zebra checkpoints. More information on how to do this can be found in [the zebra-checkpoints README](https://github.com/ZcashFoundation/zebra/blob/main/zebra-consensus/src/checkpoint/README.md).

To do this you will need a synchronized `zcashd` node. You can request help from other zebra team members to submit this PR if you can't make it yourself at the moment of the release.

## Change Log

**Important**: Any merge into `main` deletes any edits to the draft changelog.
Once you are ready to tag a release, copy the draft changelog into `CHANGELOG.md`.

We use [the Release Drafter workflow](https://github.com/marketplace/actions/release-drafter) to automatically create a [draft changelog](https://github.com/ZcashFoundation/zebra/releases).

We follow the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.

To create the final change log:
- [x] Copy the **latest** draft changelog into `CHANGELOG.md` (there can be multiple draft releases)
- [x] Delete any trivial changes. Keep the list of those, to include in the PR
- [x] Combine duplicate changes
- [x] Edit change descriptions so they are consistent, and make sense to non-developers
- [x] Check the category for each change
  - Prefer the "Fix" category if you're not sure

<details>

#### Change Categories

From "Keep a Changelog":
* `Added` for new features.
* `Changed` for changes in existing functionality.
* `Deprecated` for soon-to-be removed features.
* `Removed` for now removed features.
* `Fixed` for any bug fixes.
* `Security` in case of vulnerabilities.

</details>

### Create the Release PR

After you have the version increments, the updated checkpoints and the updated changelog:

- [x] Make sure the PR with the new checkpoint hashes is already merged, or make it part of the changelog PR
- [x] Push the version increments and the updated changelog into a branch
      (for example: `bump-v1.0.0-rc.0` - this needs to be different to the tag name)
- [x] Create a release PR by adding `&template=release-checklist.md` to the
      comparing url ([Example](https://github.com/ZcashFoundation/zebra/compare/v1.0.0-rc.0-release?expand=1&template=release-checklist.md)).
  - [x] Add the list of deleted changelog entries as a comment to make reviewing easier.
- [x] Turn on [Merge Freeze](https://www.mergefreeze.com/installations/3676/branches).
- [x] Once the PR is ready to be merged, unfreeze it [here](https://www.mergefreeze.com/installations/3676/branches).
      Do not unfreeze the whole repository.
- [x] Update the PR to the latest `main` branch using `@mergifyio update`. Then Mergify should merge it in-place.
      If it makes a merge PR instead, that PR will get cancelled by the merge freeze. So just merge the changelog PR manually.

### Create the Release

- [x] Once the PR has been merged, create a new release using the draft release as a base,
      by clicking the Edit icon in the [draft release](https://github.com/ZcashFoundation/zebra/releases)
- [x] Set the tag name to the version tag,
      for example: `v1.0.0-rc.0`
- [x] Set the release to target the `main` branch
- [x] Set the release title to `Zebra ` followed by the version tag,
      for example: `Zebra 1.0.0-rc.0`
- [x] Replace the prepopulated draft changelog in the release description by the final
      changelog you created; starting just _after_ the title `## [Zebra ...` of
      the current version being released, and ending just _before_ the title of
      the previous release.
- [x] Mark the release as 'pre-release', until it has been built and tested
- [x] Publish the pre-release to GitHub using "Publish Release"
- [x] Delete all the [draft releases from the list of releases](https://github.com/ZcashFoundation/zebra/releases)

## Binary Testing

- [x] Wait until the [Docker binaries have been built on `main`](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml), and the quick tests have passed.
      (You can ignore the full sync and `lightwalletd` tests, because they take about a day to run.)
- [x] [Publish the release to GitHub](https://github.com/ZcashFoundation/zebra/releases) by disabling 'pre-release', then clicking "Set as the latest release"
- [x] Wait until [the Docker images have been published](https://github.com/ZcashFoundation/zebra/actions/workflows/release-binaries.yml)
- [x] Test the Docker image using `docker run --tty --interactive zfnd/zebra:1.0.0-rc.<version>` 
- [x] Turn off [Merge Freeze](https://www.mergefreeze.com/installations/3676/branches) for the whole repository

## Blog Post

If the release contains new features (`major` or `minor`), or high-priority bug fixes:
- [ ] Ask the team about doing a blog post

If building or running fails after tagging:
1. Fix the bug that caused the failure
2. Increment versions again, following these instructions from the start
3. Update the code and documentation with a **new** git tag
4. Update `CHANGELOG.md` with details about the fix
5. Tag a **new** release

### CODE REVIEW

**Comment by teor2345** [comment]
Here are the deleted changelog entries, I made them unique using `sort -u`.

```
- Fix typo ([#5886](https://github.com/ZcashFoundation/zebra/pull/5886))
- build(deps): bump actions/checkout from 3.1.0 to 3.2.0 ([#5855](https://github.com/ZcashFoundation/zebra/pull/5855))
- build(deps): bump actions/checkout from 3.2.0 to 3.3.0 ([#5918](https://github.com/ZcashFoundation/zebra/pull/5918))
- build(deps): bump inferno from 0.11.12 to 0.11.13, and allow a duplicate dependency ([#5798](https://github.com/ZcashFoundation/zebra/pull/5798))
- build(deps): bump insta from 1.21.2 to 1.22.0 ([#5790](https://github.com/ZcashFoundation/zebra/pull/5790))
- build(deps): bump insta from 1.22.0 to 1.23.0 ([#5847](https://github.com/ZcashFoundation/zebra/pull/5847))
- build(deps): bump insta from 1.23.0 to 1.24.1 ([#5913](https://github.com/ZcashFoundation/zebra/pull/5913))
- build(deps): bump num\_cpus from 1.14.0 to 1.15.0 ([#5888](https://github.com/ZcashFoundation/zebra/pull/5888))
- build(deps): bump once\_cell from 1.16.0 to 1.17.0 ([#5903](https://github.com/ZcashFoundation/zebra/pull/5903))
- build(deps): bump owo-colors from 3.5.0 to 3.6.0 ([#5897](https://github.com/ZcashFoundation/zebra/pull/5897))
- build(deps): bump prost from 0.11.3 to 0.11.5 ([#5889](https://github.com/ZcashFoundation/zebra/pull/5889))
- build(deps): bump rayon from 1.6.0 to 1.6.1 ([#5845](https://github.com/ZcashFoundation/zebra/pull/5845))
- build(deps): bump reviewdog/action-actionlint from 1.34.2 to 1.35.0 ([#5856](https://github.com/ZcashFoundation/zebra/pull/5856))
- build(deps): bump rlimit from 0.8.3 to 0.9.0 ([#5900](https://github.com/ZcashFoundation/zebra/pull/5900))
- build(deps): bump semver from 1.0.14 to 1.0.16 ([#5878](https://github.com/ZcashFoundation/zebra/pull/5878))
- build(deps): bump serde from 1.0.148 to 1.0.149 ([#5788](https://github.com/ZcashFoundation/zebra/pull/5788))
- build(deps): bump serde from 1.0.149 to 1.0.150 ([#5846](https://github.com/ZcashFoundation/zebra/pull/5846))
- build(deps): bump serde from 1.0.150 to 1.0.151 ([#5877](https://github.com/ZcashFoundation/zebra/pull/5877))
- build(deps): bump serde from 1.0.151 to 1.0.152 ([#5898](https://github.com/ZcashFoundation/zebra/pull/5898))
- build(deps): bump serde\_json from 1.0.89 to 1.0.91 ([#5879](https://github.com/ZcashFoundation/zebra/pull/5879))
- build(deps): bump shimataro/ssh-key-action from 2.4.0 to 2.5.0 ([#5896](https://github.com/ZcashFoundation/zebra/pull/5896))
- build(deps): bump thiserror from 1.0.37 to 1.0.38 ([#5880](https://github.com/ZcashFoundation/zebra/pull/5880))
- build(deps): bump tj-actions/changed-files from 34.5.0 to 34.5.1 ([#5800](https://github.com/ZcashFoundation/zebra/pull/5800))
- build(deps): bump tj-actions/changed-files from 34.5.1 to 34.5.4 ([#5857](https://github.com/ZcashFoundation/zebra/pull/5857))
- build(deps): bump tj-actions/changed-files from 34.5.4 to 34.6.1 ([#5868](https://github.com/ZcashFoundation/zebra/pull/5868))
- build(deps): bump tj-actions/changed-files from 34.6.1 to 35.0.1 ([#5875](https://github.com/ZcashFoundation/zebra/pull/5875))
- build(deps): bump tj-actions/changed-files from 35.0.1 to 35.1.0 ([#5885](https://github.com/ZcashFoundation/zebra/pull/5885))
- build(deps): bump tj-actions/changed-files from 35.1.0 to 35.2.0 ([#5902](https://github.com/ZcashFoundation/zebra/pull/5902))
- build(deps): bump tj-actions/changed-files from 35.2.0 to 35.2.1 ([#5905](https://github.com/ZcashFoundation/zebra/pull/5905))
- build(deps): bump tj-actions/changed-files from 35.2.1 to 35.3.1 ([#5919](https://github.com/ZcashFoundation/zebra/pull/5919))
- build(deps): bump tokio from 1.22.0 to 1.23.0 ([#5802](https://github.com/ZcashFoundation/zebra/pull/5802))
- build(deps): bump tokio from 1.23.0 to 1.23.1 ([#5920](https://github.com/ZcashFoundation/zebra/pull/5920))
- build(deps): bump tokio from 1.23.1 to 1.24.1 ([#5927](https://github.com/ZcashFoundation/zebra/pull/5927))
- build(deps): bump toml from 0.5.9 to 0.5.10 ([#5864](https://github.com/ZcashFoundation/zebra/pull/5864))
- build(deps): bump vergen from 7.4.3 to 7.4.4 ([#5893](https://github.com/ZcashFoundation/zebra/pull/5893))
- build(deps): bump vergen from 7.4.4 to 7.5.0 ([#5921](https://github.com/ZcashFoundation/zebra/pull/5921))
- build(deps): bump w9jds/firebase-action from 11.16.0 to 11.18.0 ([#5876](https://github.com/ZcashFoundation/zebra/pull/5876))
- build(deps): bump zcash\_primitives from 0.9.0 to 0.9.1 ([#5801](https://github.com/ZcashFoundation/zebra/pull/5801))
- change(doc): Update release, ticket, and PR templates ([#5793](https://github.com/ZcashFoundation/zebra/pull/5793))
- change(rpc): Refactor get block template RPC into stages ([#5837](https://github.com/ZcashFoundation/zebra/pull/5837))
- change(rpc): Update ZIP-317 transaction selection algorithm ([#5776](https://github.com/ZcashFoundation/zebra/pull/5776))
- change(rpc): return u64 from `get_network_sol_ps` and remove `arbitrary_precision` feature from serde ([#5829](https://github.com/ZcashFoundation/zebra/pull/5829))
- docs(release): update GitHub release promoting language ([#5810](https://github.com/ZcashFoundation/zebra/pull/5810))
- fix(ci): Fix cargo deny syntax ([#5806](https://github.com/ZcashFoundation/zebra/pull/5806))
- fix(ci): Run basic and ignored unit tests at the same time, speeding up CI by half an hour ([#5814](https://github.com/ZcashFoundation/zebra/pull/5814))
- fix(clippy): Fix nightly clippy redundant code lints ([#5916](https://github.com/ZcashFoundation/zebra/pull/5916))
- fix(clippy): Put Rust format variables inline ([#5783](https://github.com/ZcashFoundation/zebra/pull/5783))
- fix(dev): Make zcash-rpc-diff always check Zebra then zcashd ([#5822](https://github.com/ZcashFoundation/zebra/pull/5822))
- fix(lint): Box large error types to resolve the clippy large result err variant lint ([#5759](https://github.com/ZcashFoundation/zebra/pull/5759))
- fix(nightly): Remove redundant `track_caller` which now causes errors on nightly ([#5839](https://github.com/ZcashFoundation/zebra/pull/5839))
- fix(test): Avoid spurious rpc\_getblocktemplate test failure ([#5909](https://github.com/ZcashFoundation/zebra/pull/5909))
- fix(test): Fix a test child process panic bug ([#5842](https://github.com/ZcashFoundation/zebra/pull/5842))
- fix(test): Move test-only code to test-only modules ([#5777](https://github.com/ZcashFoundation/zebra/pull/5777))
- fix(test): Reduce verbose test logs ([#5825](https://github.com/ZcashFoundation/zebra/pull/5825))
- refactor(state): Make implementation of block consensus rules clearer ([#5915](https://github.com/ZcashFoundation/zebra/pull/5915))
```

File: zebrad/Cargo.toml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+repository = "https://github.com/ZcashFoundation/zebra"
+
+# Settings that impact compilation
 edition = "2021"
+
 # Zebra is only supported on the latest stable Rust version. Some earlier versions might work.
-# Zebra uses features introduced in Rust 1.58.
-rust-version = "1.58"
-repository = "https://github.com/ZcashFoundation/zebra"
+# Zebra currently uses features introduced in Rust 1.65 or later.
```
Comment (arya2) [inline zebrad/Cargo.toml]: Nitpick:
```suggestion
# Zebra currently uses features included in Rust 1.65 or later.
```

**Review by arya2 — DISMISSED** [review]
The version updates and changelog look good to me.

File: zebrad/Cargo.toml (hunk 1 above)
Comment (teor2345) [reply zebrad/Cargo.toml]: Hmm, I'm not sure that's what I want to say here.

I want to say something like "we use a Rust feature that was introduced in 1.65, so Zebra won't compile with earlier versions. We might also use later features as soon as they are in stable, so this isn't a minimum supported Rust version, it's just a guideline."

Comment (arya2) [reply zebrad/Cargo.toml]: > I want to say something like "we use a Rust feature that was introduced in 1.65, so Zebra won't compile with earlier versions. We might also use later features as soon as they are in stable, so this isn't a minimum supported Rust version, it's just a guideline."

That sounds better, maybe "Zebra currently uses features introduced in Rust 1.65, and may later use features from the latest stable release of Rust."?

Comment (teor2345) [reply zebrad/Cargo.toml]: Sounds good, I'll apply that change throughout this PR.

Comment (teor2345) [reply zebrad/Cargo.toml]: What do you think of 66af9819b9?

I tried to make it a bit shorter.

Comment (arya2) [reply zebrad/Cargo.toml]: I think that's good now without the "currently", in principle Zebra uses features introduced in any later stable releases. 👍

**Review by arya2 — APPROVED** [review]

**Comment by teor2345** [comment]
The docker image seems to work fine:
```
$ docker run --tty --interactive zfnd/zebra:1.0.0-rc.3                                                                                                                        [171/2393]Unable to find image 'zfnd/zebra:1.0.0-rc.3' locally                                                 
1.0.0-rc.3: Pulling from zfnd/zebra                                                                                                                                                                        
3f4ca61aafcd: Pull complete                                                                                                                                                                                dad627728526: Pull complete                                                                                                                                                                                
2a647d9274ed: Pull complete                                                                                                                                                                                
1c7dd091ba4b: Pull complete                                                                                                                                                                                0c636a371c92: Pull complete                                                                          
5814efdbf87f: Pull complete                                                                                                                                                                                
Digest: sha256:af189a47e53fb33bf49be0c5b6bd9133992b4fc238cefe9968601066aa55e43e                                                                                                                            Status: Downloaded newer image for zfnd/zebra:1.0.0-rc.3                                                                                                                                                   
[sentry] initialized disabled sentry client due to disabled or invalid DSN                                                                                                                                 
2023-01-10T08:03:59.434200Z  INFO zebrad::components::tracing::component: started tracing component filter="info" TRACING_STATIC_MAX_LEVEL=LevelFilter::INFO LOG_STATIC_MAX_LEVEL=Info                     2023-01-10T08:03:59.434329Z  INFO zebrad::components::tracing::component: installed sentry tracing layer                                                                                           
2023-01-10T08:03:59.434337Z  INFO zebrad::application: Diagnostic Metadata:                                                                                                                                version: 1.0.0-rc.3                                                                                                                                                                                        
Zcash network: Mainnet                                                                                                                                                                                     
state version: 25                                                                                                                                                                                          target triple: x86_64-unknown-linux-gnu                                                                                                                                                                    
build profile: release                                                                                                                                                                                     
2023-01-10T08:03:59.434339Z  INFO zebrad::application: loaded zebrad config config_path=Some("/etc/zebra/zebrad.toml") config=ZebradConfig { consensus: Config { checkpoint_sync: true, debug_skip_parameter_preload: false }, metrics: Config { endpoint_addr: None }, network: Config { listen_addr: 0.0.0.0:8233, network: Mainnet, initial_mainnet_peers: {"dnsseed.z.cash:8233", "dnsseed.str4d.xyz:8233", "mainn
et.seeder.zfnd.org:8233", "mainnet.is.yolo.money:8233"}, initial_testnet_peers: {"dnsseed.testnet.z.cash:18233", "testnet.seeder.zfnd.org:18233", "testnet.is.yolo.money:18233"}, peerset_initial_target_si
ze: 25, crawl_new_peer_interval: 61s }, state: Config { cache_dir: "/zebrad-cache", ephemeral: false, debug_stop_at_height: None, delete_old_database: true }, tracing: Config { use_color: true, force_use_color: false, filter: None, buffer_limit: 128000, endpoint_addr: None, flamegraph: None, use_journald: false }, sync: Config { download_concurrency_limit: 50, checkpoint_verify_concurrency_limit: 1000, 
full_verify_concurrency_limit: 20, parallel_cpu_threads: 0 }, mempool: Config { tx_cost_limit: 80000000, eviction_memory_time: 3600s, debug_enable_at_height: None }, rpc: Config { listen_addr: None, parallel_cpu_threads: 0, debug_force_finished_sync: false } }                                                                                                                                                  
2023-01-10T08:03:59.434916Z  INFO {net="Main"}: zebrad::application: initialized rayon thread pool for CPU-bound tasks num_threads=36                                                                      
2023-01-10T08:03:59.436144Z  INFO {net="Main"}: zebrad::commands::start: Starting zebrad                                                                                                                   2023-01-10T08:03:59.436426Z  INFO {net="Main"}: zebrad::commands::start: initializing node state                                                                                                           
2023-01-10T08:03:59.441192Z  INFO {net="Main"}: zebrad::commands::start: opening database, this may take a few minutes                                                                                     
2023-01-10T08:03:59.442094Z  INFO zebra_state::service::finalized_state::disk_db: the open file limit is high enough for Zebra current_limit=1048576 min_limit=512 ideal_limit=1024                        2023-01-10T08:03:59.508597Z  INFO zebra_state::service::finalized_state::disk_db: Opened Zebra state cache at /zebrad-cache/state/v25/mainnet                                                              
2023-01-10T08:03:59.509038Z  INFO zebra_state::service::finalized_state: loaded Zebra state cache tip=None                                                                                                 2023-01-10T08:03:59.509279Z  INFO zebra_state::service: starting legacy chain check                                                                                                                        
2023-01-10T08:03:59.509301Z  INFO zebra_state::service: cached state consensus branch is valid: no legacy chain found                                                                                      
2023-01-10T08:03:59.509409Z  INFO {net="Main"}: zebrad::commands::start: initializing network                                                                                                              2023-01-10T08:03:59.509457Z  INFO {net="Main"}:open_listener{addr=0.0.0.0:8233}: zebra_network::peer_set::initialize: Trying to open Zcash protocol endpoint at 0.0.0.0:8233...                     
2023-01-10T08:03:59.509503Z  INFO {net="Main"}:open_listener{addr=0.0.0.0:8233}: zebra_network::peer_set::initialize: Opened Zcash protocol endpoint at 0.0.0.0:8233                                       
2023-01-10T08:03:59.509593Z  INFO {net="Main"}: zebra_network::address_book_updater: starting the address book updater                                                                                     2023-01-10T08:04:00.157396Z  INFO {net="Main"}:add_initial_peers: zebra_network::config: resolved seed peer IP addresses seed="dnsseed.z.cash:8233" remote_ip_count=25                              
2023-01-10T08:04:00.224205Z  INFO {net="Main"}:add_initial_peers: zebra_network::config: resolved seed peer IP addresses seed="mainnet.is.yolo.money:8233" remote_ip_count=4                               
2023-01-10T08:04:00.331207Z  INFO {net="Main"}:add_initial_peers: zebra_network::config: resolved seed peer IP addresses seed="dnsseed.str4d.xyz:8233" remote_ip_count=25                                  2023-01-10T08:04:04.510498Z  INFO {net="Main"}:add_initial_peers: zebra_network::config: DNS timeout resolving peer IP addresses host="mainnet.seeder.zfnd.org:8233" e=Elapsed(())                         
2023-01-10T08:04:04.510621Z  INFO {net="Main"}:add_initial_peers: zebra_network::config: Retrying peer DNS resolution host="mainnet.seeder.zfnd.org:8233" retry_count=1                                    2023-01-10T08:04:09.512111Z  INFO {net="Main"}:add_initial_peers: zebra_network::peer_set::initialize: limiting the initial peers list from 48 to 25                                                       
2023-01-10T08:04:09.512412Z  INFO {net="Main"}:add_initial_peers: zebra_network::peer_set::initialize: connecting to initial peer set initial_peer_count=25 initial_peers={39.97.178.10:8233, 95.217.78.170
:8233, 136.243.145.143:8233, 47.254.176.240:8233, 51.79.57.29:8233, [2a01:4f8:140:2393::2]:8233, 91.199.137.99:8233, 8.210.73.119:8233, 39.97.172.77:8233, 165.232.125.107:8233, 82.121.36.202:8233, 37.187.88.208:8233, 37.128.74.73:8233, 92.154.55.207:8233, 95.217.108.172:8233, [2a01:4f8:b0:920a::5502]:8233, 64.227.136.170:8233, 47.198.223.60:8233, 15.235.85.30:8233, 168.119.147.118:8233, 104.233.147.162:
8233, 188.166.173.215:8233, 39.97.242.143:8233, 135.148.55.16:8233, 157.245.172.190:8233}                                                                                                                  
2023-01-10T08:04:12.589834Z  INFO {net="Main"}:add_initial_peers: zebra_network::peer_set::initialize: an initial peer connection failed successes=20 errors=3 addr=47.254.176.240:8233 e=request timed out2023-01-10T08:04:12.714327Z  INFO {net="Main"}:add_initial_peers: zebra_network::peer_set::initialize: an initial peer connection failed successes=20 errors=4 addr=39.97.172.77:8233 e=request timed out  
2023-01-10T08:04:12.915081Z  INFO {net="Main"}:add_initial_peers: zebra_network::peer_set::initialize: an initial peer connection failed successes=20 errors=5 addr=64.227.136.170:8233 e=request timed out
2023-01-10T08:04:12.915120Z  INFO {net="Main"}:add_initial_peers: zebra_network::peer_set::initialize: finished connecting to initial seed peers handshake_success_total=20 handshake_error_total=5 outbound_connections=22                                                                                                                                                                                           
2023-01-10T08:04:12.915390Z  INFO {net="Main"}: zebra_network::peer_set::initialize: sending initial request for peers active_initial_peer_count=22                                                        2023-01-10T08:04:12.916174Z  INFO {net="Main"}: zebrad::commands::start: initializing verifiers                                                                                                            
2023-01-10T08:04:12.916205Z  INFO {net="Main"}:crawl_and_dial{new_peer_interval=61s}: zebra_network::peer_set::initialize: starting the peer address crawler crawl_new_peer_interval=61s outbound_connectio
ns=22
2023-01-10T08:04:12.916482Z  INFO {net="Main"}:init{config=Config { checkpoint_sync: true, debug_skip_parameter_preload: false } network=Mainnet debug_skip_parameter_preload=false}: zebra_consensus::prim
itives::groth16::params: checking and loading Zcash Sapling and Sprout parameters
2023-01-10T08:04:12.919841Z  INFO {net="Main"}:init{config=Config { checkpoint_sync: true, debug_skip_parameter_preload: false } network=Mainnet debug_skip_parameter_preload=false}: zebra_consensus::chai
n: initializing chain verifier tip=None max_checkpoint_height=Height(1941759)
2023-01-10T08:04:12.919879Z  INFO {net="Main"}: zebrad::commands::start: initializing syncer
2023-01-10T08:04:12.919915Z  INFO {net="Main"}: zebrad::commands::start: initializing mempool
2023-01-10T08:04:12.919975Z  INFO {net="Main"}: zebrad::commands::start: spawned initial Zebra tasks
2023-01-10T08:04:12.919999Z  INFO {net="Main"}: zebra_state::config: checking for old database versions
2023-01-10T08:04:12.920068Z  INFO {net="Main"}: zebrad::components::sync::gossip: initializing block gossip task
2023-01-10T08:04:12.920071Z  INFO {net="Main"}: zebrad::components::mempool::crawler: initializing mempool crawler task
2023-01-10T08:04:12.920099Z  INFO {net="Main"}: zebrad::components::mempool::queue_checker: initializing mempool queue checker task
2023-01-10T08:04:12.920107Z  INFO {net="Main"}: zebrad::components::mempool::gossip: initializing transaction gossip task
2023-01-10T08:04:12.920488Z  INFO {net="Main"}:sync: zebrad::components::sync: starting genesis block download and verify
2023-01-10T08:04:12.922801Z  INFO {net="Main"}: zebra_state::config: finished old database version cleanup task
2023-01-10T08:04:12.923604Z  INFO {net="Main"}: zebrad::components::sync::progress: initial sync is waiting to download the genesis block sync_percent=0.000 % current_height=None
2023-01-10T08:04:13.086018Z  INFO {net="Main"}:sync:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=1 current_range=(Unbounded, Included(Height(0)))
2023-01-10T08:04:13.150304Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: starting sync, obtaining new tips state_tip=Some(Height(0))
2023-01-10T08:04:13.747493Z  INFO {net="Main"}:init{config=Config { checkpoint_sync: true, debug_skip_parameter_preload: false } network=Mainnet debug_skip_parameter_preload=false}: zebra_consensus::chai
n: Groth16 pre-download and check task finished
2023-01-10T08:04:17.729970Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=499 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(0))
2023-01-10T08:04:22.356422Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=997 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(0))
2023-01-10T08:04:29.003750Z  INFO {net="Main"}:sync:try_to_sync:obtain_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=800 current_range=(Excluded(Height(0)), Included
(Height(800)))
2023-01-10T08:04:32.262558Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(800))
2023-01-10T08:04:37.502337Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=800 current_range=(Excluded(Height(800)), Includ
ed(Height(1600)))
2023-01-10T08:04:37.550191Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(994))
2023-01-10T08:04:38.324490Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=400 current_range=(Excluded(Height(1600)), Inclu
ded(Height(2000)))
2023-01-10T08:04:41.763333Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(2000))
2023-01-10T08:04:46.222212Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(2000))
2023-01-10T08:04:52.051795Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=400 current_range=(Excluded(Height(2000)), Inclu
ded(Height(2400)))
2023-01-10T08:04:58.712978Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=800 current_range=(Excluded(Height(2400)), Inclu
ded(Height(3200)))
2023-01-10T08:04:58.782451Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(2487))
2023-01-10T08:05:01.215558Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=400 current_range=(Excluded(Height(3200)), Inclu
ded(Height(3600)))
2023-01-10T08:05:03.338156Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(3600))
2023-01-10T08:05:08.009303Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(3600))
2023-01-10T08:05:12.924477Z  INFO {net="Main"}: zebrad::components::sync::progress: estimated progress to chain tip sync_percent=0.184% current_height=Height(3600) network_upgrade=BeforeOverwinter remain
ing_sync_blocks=1952659 time_since_last_state_block=0s
2023-01-10T08:05:15.604140Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=1200 current_range=(Excluded(Height(3600)), Incl
uded(Height(4800)))
2023-01-10T08:05:16.040219Z  INFO {net="Main"}:sync:try_to_sync: zebrad::components::sync: extending tips tips.len=1 in_flight=999 extra_hashes=0 lookahead_limit=1000 state_tip=Some(Height(3982))
2023-01-10T08:05:19.408945Z  INFO {net="Main"}:sync:try_to_sync:extend_tips:checkpoint: zebra_consensus::checkpoint: verified checkpoint range block_count=400 current_range=(Excluded(Height(4800)), Included(Height(5200)))
...
```

**Pull request merged**